_includes/custom-head.html

```diff
@@ -7,18 +7,18 @@
 
 
 {%- include favicons.html -%}
-<link href="https://unpkg.com/@primer/css/dist/primer.css" rel="stylesheet" />
-<link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+<link href="https://cdn.jsdelivr.net/npm/@primer/css/dist/primer.css" rel="stylesheet" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.css">
 
 {%- if site.annotations -%}
-<script src="https://hypothes.is/embed.js" async></script>
+<script src="https://cdn.jsdelivr.net/npm/hypothesis/build/boot.js" async></script>
```

Comment:
I would rather stick with the source listed in [the official docs](https://web.hypothes.is/for-publishers/#embedding).

Reply:
I can't open the web.hypothes.is website.

Reply:
@illcat where are you located?  I'm weary of hosting arbitrary sites I am not familiar with as it is an attack vector, so I have an aversion towards straying from the official docs

Reply:
China